app/build.gradle.kts

```diff
@@ -147,10 +147,10 @@ dependencies {
     implementation("com.maltaisn:iconpack-community-material:5.3.45")
 
     implementation("org.jetbrains.kotlin:kotlin-stdlib:1.8.21")
-    implementation("org.jetbrains.kotlin:kotlin-reflect:1.8.10")
+    implementation("org.jetbrains.kotlin:kotlin-reflect:1.8.21")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.7.1")
     implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.7.1")
-    "fullImplementation"("org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.6.4")
+    "fullImplementation"("org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.7.1")
```

Comment:
isnt this duplicated by line 176?

Reply:
Yes but removing it breaks the automotive build because it doesn't do full/minimal properly, see the open PR for that 😅. I decided to just keep it for now as we'll likely pick it up when reconfiguring automotive for full/minimal?

Reply:
ooh I see what you mean now :)